stubs/jsonschema/jsonschema/exceptions.pyi

```diff
@@ -72,11 +72,12 @@ class FormatError(Exception):
     def __init__(self, message, cause: Incomplete | None = None) -> None: ...
 
 class ErrorTree:
-    errors: Incomplete
-    def __init__(self, errors=()) -> None: ...
-    def __contains__(self, index): ...
+    errors: MutableMapping[str, ValidationError]
+    def __init__(self, errors: Iterable[ValidationError] = ()) -> None: ...
+    def __contains__(self, index: str | int) -> bool: ...
```

Comment:
I know this is just copied from the runtime package's source code, but this means that type checkers won't see this class as a subtype of `Container`, since `Container.__contains__` has `object` as the parameter annotation. So it's questionable whether the annotation in the runtime package is correct.

Reply:
Looking at the source, `object` should work fine here.